### PR TITLE
[master] Porting #52670 to master

### PR DESCRIPTION
**Pull request by garethgreenaway**
Porting #52670 to master

### CODE REVIEW

**Review by dwoz — APPROVED** [review]

File: salt/modules/debian_ip.py [hunk 1]
```diff
@@ -1808,7 +1808,8 @@ def down(iface, iface_type):
     # Slave devices are controlled by the master.
     # Source 'interfaces' aren't brought down.
     if iface_type not in ['slave', 'source']:
-        return __salt__['cmd.run'](['ifdown', iface])
+        cmd = ['ip', 'link', 'set', '{0}'.format(iface), 'down']
```
Comment (s0undt3ch) [inline salt/modules/debian_ip.py]: Why do we `.format()` `iface`? is it ever not going to be a string?

File: salt/modules/debian_ip.py [hunk 2]
```diff
@@ -1869,7 +1870,8 @@ def up(iface, iface_type):  # pylint: disable=C0103
     # Slave devices are controlled by the master.
     # Source 'interfaces' aren't brought up.
     if iface_type not in ('slave', 'source'):
-        return __salt__['cmd.run'](['ifup', iface])
+        cmd = ['ip', 'link', 'set', '{0}'.format(iface), 'up']
```
Comment (s0undt3ch) [inline salt/modules/debian_ip.py]: Same as above.

**Review by s0undt3ch — CHANGES_REQUESTED** [review]

**Review by DmitryKuzmenko — APPROVED** [review]

File: salt/modules/debian_ip.py (hunk 2 above)
Comment (DmitryKuzmenko) [reply salt/modules/debian_ip.py]: Fixed

**Comment by DmitryKuzmenko** [comment]
@s0undt3ch I've updated the code, could you please approve this?

**Review by s0undt3ch — APPROVED** [review]

**Pull request merged**